pulp_container/app/registry_api.py

```diff
@@ -607,18 +606,16 @@ def get(self, request):
                     tag.name, tag.tagged_manifest, req_oss, req_architectures, manifests
                 )
             for manifest, tagged in manifests.items():
-                with storage.open(manifest.config_blob._artifacts.get().file.name) as file:
-                    raw_data = file.read()
-                config_data = json.loads(raw_data)
-                labels = config_data.get("config", {}).get("Labels")
+                config_data = manifest.config
```

Comment:
```suggestion
                config = manifest.config
```
Please, use correct naming conventions.

Note that this will not work in case the user has not run the django-management command to migrate artifacts.

We need to support two code paths -- for users who have run the django-management command and those who have not run it yet.